packages/advisor-components/src/RuleDetails/RuleDetails.tsx

```diff
@@ -0,0 +1,247 @@
+        return isDetailsPage ? ruleDescription(rule.generic, true) : ruleDescription(rule.summary);
+    }
+  };
+
+  const renderViewAffected = () => {
+    switch (product) {
+      case AdvisorProduct.ocp:
+        if ((rule as RuleContentOcp).impacted_clusters_count > 0) {
+          return (
+            <StackItem>
+              <Link key={`${rule.rule_id}-link`} onClick={() => onViewAffectedClick && onViewAffectedClick(rule.rule_id)} to="#">
+                {intl.formatMessage(messages.viewAffectedClusters, {
+                  clusters: (rule as RuleContentOcp).impacted_clusters_count,
+                })}
+              </Link>
+            </StackItem>
+          );
+        }
+        break;
+      case AdvisorProduct.rhel:
+        if ((rule as RuleContentRhel).impacted_systems_count > 0) {
+          return (
+            <StackItem>
+              <Link key={`${rule.rule_id}-link`} onClick={() => onViewAffectedClick && onViewAffectedClick(rule.rule_id)} to="#">
+                {intl.formatMessage(messages.viewAffectedSystems, {
+                  systems: (rule as RuleContentRhel).impacted_systems_count,
+                })}
+              </Link>
+            </StackItem>
+          );
+        }
+    }
+  };
+
+  const renderImpactSeverityLine = () => {
+    switch (product) {
+      case AdvisorProduct.ocp:
+        return (
+          <SeverityLine
+            className="severity-line"
+            title={intl.formatMessage(messages.impactLevel, {
+              level: intl.formatMessage(messages[IMPACT_LABEL_KEY[(rule as RuleContentOcp).impact]]),
+            })}
+            value={(rule as RuleContentOcp).impact}
+            tooltipMessage={intl.formatMessage(messages.impactDescription, {
+              level: intl.formatMessage(messages[IMPACT_LABEL_KEY[(rule as RuleContentOcp).impact]]).toLowerCase(),
+            })}
+          />
+        );
+        break;
+      case AdvisorProduct.rhel:
+        return (
+          <SeverityLine
+            className="severity-line"
+            title={intl.formatMessage(messages.impactLevel, {
+              level: intl.formatMessage(messages[IMPACT_LABEL_KEY[(rule as RuleContentRhel).impact.impact]]),
+            })}
+            value={(rule as RuleContentRhel).impact.impact}
+            tooltipMessage={intl.formatMessage(messages.impactDescription, {
+              level: intl.formatMessage(messages[IMPACT_LABEL_KEY[(rule as RuleContentRhel).impact.impact]]).toLowerCase(),
+            })}
+          />
+        );
+    }
+  };
+
+  return (
+    <Split className="ins-c-rule-details__split" hasGutter>
+      <SplitItem>
+        <Stack hasGutter>
+          {header && <StackItem>{header}</StackItem>}
+          <StackItem>{renderDescription()}</StackItem>
+          {(rule as RuleContentRhel).node_id && (
+            <StackItem>
+              <a rel="noopener noreferrer" target="_blank" href={`https://access.redhat.com/node/${(rule as RuleContentRhel).node_id}`}>
```

Comment:
I'd rather see the full `href` being passed from parent component. When the managers call for change of URL we'd have to change it here, release new version, use this new version and release the app itself.

Reply:
done in https://github.com/RedHatInsights/frontend-components/pull/1493/commits/e9eddcf5f5ce4ad8d15b8129808cb563f8103ef2